app/controllers/initiatives_controller.rb

```diff
@@ -1,13 +1,24 @@
 # frozen_string_literal: true
 
 class InitiativesController < ApplicationController
-  before_action :find_initiative, only: [:edit, :update, :destroy]
+  before_action :find_initiative, only: [:edit, :show, :update, :destroy]
   before_action :authenticate_user!
+  # before_action :find_team_member_intiative, only: [:show]
 
-  def index
+  def index  
     populate_initiative_sets
   end
 
+  def team_members_initiatives
+    @team_member = User.find(params[:user_id])
+    @completed_initiatives = @team_member.initiatives.completed
```

Comment:
Could this and the next line seem like they could be using #populate_initiative_sets instead?

Reply:
Resolved!